aiida/orm/querybuilder.py

```diff
@@ -79,8 +82,20 @@ class PathItemType(TypedDict):
         joining_value: str
         outerjoin: bool
         edge_tag: str
+
+    class QueryDict(TypedDict, total=False):
+        """A JSON serialisable representation of a ``QueryBuilder`` instance"""
+
+        path: List[PathItemType]
+        filters: Dict[str, FilterType]
+        project: Dict[str, ProjectType]
+        order_by: List[str]
```

Comment:
Almost sure this is a list of dicts... maybe re-check all of these to be 100 % sure?